MEMBER_REPRESENTATION.md

```diff
@@ -0,0 +1,9 @@
+# Member Representation
+
+The following document lists OpenJS Foundation members and their representation to any standards organizations:
+
+* <firstname> <lastname>
+    * [TC39][]
+    * ...
```

Comment:
```suggestion
* [TC39]
  * <firstname> <lastname> (#issue)
  * ...
```

I think the indexing is the wrong way around here, and rather than focusing on which members are representing us, we should focus on which organisations we are represented in. This would also help clarify that we're enumerating representations of OpenJS in particular, not all representations by members of the Working Group.

I also added an `#issue` link to the template, for the issue requesting and approving the representation.